spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/core/cloudfoundry/GcpCloudFoundryEnvironmentPostProcessorTests.java

```diff
@@ -46,6 +46,7 @@
 			.withInitializer(context ->
 					new GcpCloudFoundryEnvironmentPostProcessor()
 							.postProcessEnvironment(context.getEnvironment(), null))
+			.withSystemProperties(new String[] {"spring.cloud.gcp.sql.instance-connection-name="})
```

Comment:
Would we need to add lines like this for every setting for the other integration tests that we move from their application-test.properties files to travis.yml?